extensions/amp-video/0.1/video-cache.js

```diff
@@ -150,6 +151,16 @@ function applySourcesToVideo(videoEl, sources, maxBitrate) {
     });
 }
 
+/**
+ * @param {!Element} videoEl
+ * @param {boolean|undefined} hasAudio
+ */
+function applyAudioInfoToVideo(videoEl, hasAudio) {
+  if (hasAudio == false) {
+    videoEl.setAttribute('noaudio', '');
+  }
```

Comment:
I think it's fine to respect the publisher provided `noaudio` even if there is audio, but plz add a strict equality check `===`